settings.gradle

```diff
@@ -377,6 +377,15 @@ include ':clock-impl'
 
 include ':sql'
 
+include ':functions'
+project(':functions').projectDir = file('java/functions')
```

Comment:
I'm not a fan of this naming, specifically the "java/" prefix.

Reply:
Hoping to have a generic name where we can dump a lot of projects into. Open to suggestions.